pkg/sdn/plugin/node.go

```diff
@@ -94,8 +93,12 @@ func (node *OsdnNode) Start() error {
 		return fmt.Errorf("Failed to set up iptables: %v", err)
 	}
 
-	var networkChanged bool
-	networkChanged, err = node.SubnetStartNode(node.mtu)
+	networkChanged, err := node.SetupSDN()
+	if err != nil {
+		return err
+	}
+
+	err = node.SubnetStartNode()
 	if err != nil {
 		return err
 	}
```

Comment:
https://github.com/danwinship/origin/blob/9df71e32e6c65fb5eed4177bfa36d14b8b84890e/pkg/sdn/plugin/node.go#L125
What happens to the pods when UpadePod() fails in case of network change? Networking for these pods will be broken. May be we can retry couple of times and log error if it didn't succeed?